src/planwise/client/utils.cljs

```diff
@@ -73,7 +73,11 @@
          format-string (str "%." decimals "f%%")]
      (gstring/format format-string percentage))))
 
-;; Collection manipulation
+;; Collection utils
+
+(defn find-by-id
+  [coll id]
+  (first (filter #(= id (:id %)) coll)))
```

Comment:
```clojure
(some #(when (= id (:id %)) %) coll)
```

should be more efficient. Or even:

```clojure
(reduce #(when (= id (:id %2)) (reduced %2)) nil coll)
```

Ref: http://stackoverflow.com/a/32405094/718209